commander.yaml

```
@@ -19,3 +19,235 @@ tests:
 
   auth0 completion bash:
     exit-code: 0
+  
+  # Test 'apps create' --type flag
+  apps create type native and check data:
+    command: auth0 apps create --name integration-test-nativeapp1 --type native --description NativeApp1 --format json
+    exit-code: 0
+    stdout:
+      json:
+        name: integration-test-nativeapp1
+        description: NativeApp1
+        app_type: native
+
+  apps create type native and check output:
+    command: auth0 apps create --name integration-test-nativeapp1 --type native --description NativeApp1
+    exit-code: 0
+    stdout:
+      contains:
+        - NAME                 integration-test-nativeapp1
+        - DESCRIPTION          NativeApp1
+        - TYPE                 Native
```

This test is failing for me due to `N` being uppercase in `contains` but lower case in the result:
```
✗ [local] 'apps create type native and check output', on property 'Stdout'

Expected
[...]
  TYPE                 native
```

```suggestion
        - TYPE                 native
```

I can see that we're already testing this command [here](https://github.com/auth0/auth0-cli/pull/278/files#diff-968aa84e08a507b385aee72591b5751aafdaac0d93109337e03ba0ba860237a5R24) but this command is the plain format. Have you considered removing this plain format test all together?

---

✋  Ignore this. Error was on my side. I was running an older version of `auth0`.

---

Removing the plain format tests that duplicate json format tests is a good question, though. I included it in case we want an indication when the formatting changes unexpectedly but it could be fussy to maintain.